src/Aspire.Hosting.Azure.AppContainers/AzureContainerAppEnvironmentResource.cs

```diff
@@ -0,0 +1,105 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Hosting.ApplicationModel;
+
+namespace Aspire.Hosting.Azure.AppContainers;
+
+/// <summary>
+/// Represents an Azure Container App Environment resource.
+/// </summary>
+/// <param name="name">The name of the Container App Environment.</param>
+/// <param name="configureInfrastructure">The callback to configure the Azure infrastructure for this resource.</param>
+public class AzureContainerAppEnvironmentResource(string name, Action<AzureResourceInfrastructure> configureInfrastructure) :
+    AzureProvisioningResource(name, configureInfrastructure), IAzureContainerAppEnvironment
+{
+    /// <summary>
+    /// Gets the unique identifier of the Container App Environment.
+    /// </summary>
+    public BicepOutputReference ContainerAppEnvironmentId => new("AZURE_CONTAINER_APPS_ENVIRONMENT_ID", this);
+
+    /// <summary>
+    /// Gets the default domain associated with the Container App Environment.
+    /// </summary>
+    public BicepOutputReference ContainerAppDomain => new("AZURE_CONTAINER_APPS_ENVIRONMENT_DEFAULT_DOMAIN", this);
+
+    /// <summary>
+    /// Gets the managed identity ID associated with the Container App Environment.
+    /// </summary>
+    public BicepOutputReference ManagedIdentityId => new("AZURE_CONTAINER_REGISTRY_MANAGED_IDENTITY_ID", this);
```

Comment:
Is this supposed to be `clientId` or `principalId`?

The output name is wrong - it is dupicated with `ContainerRegistryManagedIdentityId` below.

Reply:
It would be easier to analyze if these output references were grouped together in some coherent manner.

Reply:
So this identity is really the keyvault identity (I am going to do a pass to split the identity and improve the naming here). https://github.com/dotnet/aspire/issues/8133